core/base/src/TException.cxx

```diff
@@ -18,13 +18,19 @@ exception handling functionality.
 
 #include "TException.h"
 
-ExceptionContext_t *gException;
+#ifdef WIN32
+#define R__DLLEXPORT __declspec(dllexport)
+#else
+#define R__DLLEXPORT
+#endif
```

Comment:
This shouldn't be needed `__declspec(dllexport)` should only be used in header files. But I'll have to try to confirm